### DISCUSSION TITLE
Custom Axis Format - Change Format If Condition is Met (WPF)

### DISCUSSION THREAD

**Discussion by bstevenson-werfen** [Live-Charts/LiveCharts2#1478] [Q&A]
When we create an axis for a chart, we can set various attributes in regard to how the whole axis object can look, like color, font, etc...  This is applied to every label in the axis of the chart.

However, we have a business need to be able to change an **individual or a number of label(s)** to have a custom format when a condition is met, **not** the entire axis.

How can we achieve this using LiveCharts2?  Searching within the documentation and on this forum did not produce any results.

Thank you.

**Comment by ryll-spectre** [comment]
Same issue and need here.  Have you figured anything out?
